### ISSUE TITLE
Repeat last align

### ISSUE THREAD

**Issue by borisbrodski** [junegunn/vim-easy-align#144]
I would like to repeat last align, that was configured interactively.

After finishing easy-align interactive mode the plugin prints the resulted command and execute it.
This is very nice. But I couldn't find a way to copy this resulted command and reissue it or map it to a key stroke.

If there is currently not way of copying last issued align command, consider this a feature request.

Thank you for amazing plugin. Just love it!

**Comment by strboul** [comment]
I suppose you have to install a separate plugin for this. See the docs `:h <Plug>(EasyAlign)`

> Notice that the commands are repeatable with `.` key if you have installed
{repeat.vim}{3}. Install {visualrepeat}{4} as well if you want to repeat in
visual mode.
>
> {3} https://github.com/tpope/vim-repeat
> {4} https://github.com/vim-scripts/visualrepeat

**Comment by borisbrodski** [comment]
Thank you very much for your response. I can repeat the Align with the dot (.), no problem here. What I rather mean is to repeat visually configured align at some point in the future.

Let me rephrase my question/feature request:

* How can I map a visually configured align to some hotkey or to a command?
